share/governance.md

```diff
@@ -148,12 +155,39 @@ role. Appointments last until either the officer steps down, or the unlikely
 and unfortunate situation where the PMC removes the officer. The PMC can add
 more officer roles as it sees fit by modifying this document.
 
-1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating
-  * The election officer will be a member of the `election-officer` GitHub group which grants him write access to the plumbing repository.
+1) Election Officer: The election officer can be on the PMC, as long as they are not running in the election they are officiating.
+  * The election officer will be a member of the `election-officer` GitHub group which grants them write access to the plumbing repository.
+
+2) Security Officer: This is the point of contact for external or internal security issues, this person has a published GPG key, and will be the main point of contact for CVE numbers and such.
+
+3) Communications Officer: This is the main point of contact for external and internal publicity and marketing efforts and requests.
+
+### Subcommittees
+
+Each subcommittee will have at least one member who is also on the PMC and at
+least three members in total. To be elligible for a subcommittee, one must
+already be Collaborator.
+
+#### Module committee
+
+This committee will manage what Vox Pupuli has generally managed including
```

Comment:
It's not yet clear to me what the committee should do.

Reply:
What the current PMC does now in terms of working on puppet modules, linters, etc and the decisions around those.

Reply:
This should be fleshed out with specific tasks, my take on what those might be initially are below.

I feel that this is less about actual maintenance of the modules, but more the plumbing of that work.  So, approving modules to be moved into Vox, ensuring modules have primary maintainers, maintaining modulesync.

Reply:
It's both. My thoughts here were to capture what VP focuses on today.